docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+
+TLS::
+* Add `ChaCha20` TLS ciphers on Java 12+ {es-pull}42155[#42155]
+* Add support for `KeyStore` filters to `ssl-config` {es-pull}75407[#75407]
+* Update TLS ciphers and protocols for JDK 11 {es-pull}41808[#41808] (issues: {es-issue}38646[#38646], {es-issue}41385[#41385])
+
+Transform::
+* Prevent old beta transforms from starting {es-pull}79712[#79712]
+
+TSDB::
+* Automatically add timestamp mapper {es-pull}79136[#79136]
+* Create a coordinating node level reader for tsdb {es-pull}79197[#79197]
+* Fix TSDB shrink test in multi-version cluster {es-pull}79940[#79940] (issue: {es-issue}79936[#79936])
+* Do not allow shadowing metrics or dimensions {es-pull}79757[#79757]
+
+
+[[bug-8.0.0]]
+[float]
+=== Bug fixes
+
+Aggregations::
+* Fix BWC issues for `x_pack/usage` {es-pull}55181[#55181] (issue: {es-issue}54847[#54847])
+* Fix `DoubleBounds` null serialization {es-pull}59475[#59475]
+* Fix `TopHitsAggregationBuilder` adding duplicate `_score` sort clauses {es-pull}42179[#42179] (issue: {es-issue}42154[#42154])
+* Fix `t_test` usage stats {es-pull}54753[#54753] (issue: {es-issue}54744[#54744])
+* Throw exception if legacy interval cannot be parsed in `DateIntervalWrapper` {es-pull}41972[#41972] (issue: {es-issue}41970[#41970])
+
+Autoscaling::
+* Autoscaling use adjusted total memory {es-pull}80528[#80528] (issue: {es-issue}78750[#78750])
+
+CCR::
+* Fix `AutoFollow` version checks {es-pull}73776[#73776] (issue: {es-issue}72935[#72935])
+
+Cluster Coordination::
+* Apply cluster states in system context {es-pull}53785[#53785] (issue: {es-issue}53751[#53751])
+
+Data streams::
+* Prohibit restoring a data stream alias with a conflicting write data stream {es-pull}81217[#81217] (issue: {es-issue}80976[#80976])
+
+Distributed::
+* Introduce `?wait_for_active_shards=index-setting` {es-pull}67158[#67158] (issue: {es-issue}66419[#66419])
+* Respect `CloseIndexRequest#waitForActiveShards` in HLRC {es-pull}67374[#67374] (issues: {es-issue}67158[#67158], {es-issue}67246[#67246])
+* Fixes to task result index mapping {es-pull}50359[#50359] (issue: {es-issue}50248[#50248])
+
+Features/CAT APIs::
+* Fix cat recovery display of bytes fields {es-pull}40379[#40379] (issue: {es-issue}40335[#40335])
+
+Features/ILM+SLM::
+* Ensuring that the `ShrinkAction` does not hang if total shards per node is too low {es-pull}76732[#76732] (issue: {es-issue}44070[#44070])
+* Less verbose serialization of snapshot failure in SLM metadata {es-pull}80942[#80942] (issue: {es-issue}77466[#77466])
+
+Features/Indices APIs::
+* Fix `ComposableIndexTemplate` equals when `composed_of` is null {es-pull}80864[#80864]
+
+Features/Java High Level REST Client::
+* Fix HLRC compatibility with Java 8 {es-pull}74290[#74290] (issues: {es-issue}73910[#73910], {es-issue}74272[#74272], {es-issue}74289[#74289])
+* Avoid `StackOverflowError` due to regex alternate paths {es-pull}61259[#61259] (issue: {es-issue}60889[#60889])
```

Comment:
The HLRC has been removed from version 8, so I think this section should be replaced with something like "Remove HLRC - replaced by new Elasticsearch Java client", together with a link to the new Elasticsearch Java client.

There are also 3 other HLRC PRs in other sections of these release notes (found by doing a browser search for "hlrc").  These should be removed too as they're superseded by the overall removal.

Reply:
Good point. I changed this to:


> The Java High Level Rest Client (HLRC) has been removed and replaced by a new Elasticsearch Java client. For migration steps, refer to [Migrate from the High Level Rest Client](https://www.elastic.co/guide/en/elasticsearch/client/java-api-client/8.0/migrate-hlrc.html).